specifyweb/frontend/js_src/lib/components/FormParse/fields.ts

```diff
@@ -213,8 +215,10 @@ const processFieldType: {
         hasCloneButton: getProperty('cloneBtn')?.toLowerCase() === 'true',
         typeSearch: getProperty('name'),
         searchView: getProperty('searchView'),
-        hasAddButton:
-          (getProperty('addBtn')?.toLowerCase() ?? 'true') === 'true',
+        hasNewButton: getProperty('showNewButton')?.toLowerCase() !== 'false',
+        hasSearchButton:
+          getProperty('showSearchButton')?.toLowerCase() !== 'false',
+        hasEditButton: getProperty('showEditButton')?.toLowerCase() !== 'false',
```

Comment:
getProperty is being called with wrong arguments.
Should be `editBtn` `newBtn` `cloneBtn` `searchBtn` - carefully look at the spec again - you used the internal key names, not what is being read by the spec from the XML

```xml
   // showSearchButton is just an internal name - you can pick whatever you want
    showSearchButton: pipe(
   // searchBtn is what's actually read from the XML form from the "initialize" attribute
      syncers.xmlAttribute('initialize searchBtn', 'skip'),
      syncers.maybe(syncers.toBoolean),
      syncers.default<boolean>(true)
    ),
```